salt/modules/win_lgpo.py

```diff
@@ -4179,7 +4185,7 @@ def _regexSearchKeyValueCombo(policy_data, policy_regpath, policy_regkey):
                                b'\00;'])
         match = re.search(_thisSearch, policy_data, re.IGNORECASE)
         if match:
-            return policy_data[match.start():(policy_data.index(']', match.end())) + 1]
+            return policy_data[match.start():(policy_data.index(b']', match.end())) + 2]
```

Comment:
I'm sorry, I'm not aware of this but it would be cool to have an example or just a comment here explaining what's `+2`.